receiver/prometheusreceiver/internal/metricfamily.go

```diff
@@ -75,6 +76,17 @@ func newMetricFamily(metricName string, mc MetadataCache) MetricFamily {
 	}
 }
 
+func exportName(metricName string, metricExportNameMap map[string]string, exportPrefix string) string {
```

Comment:
Can you please add a comment to explain the purpose of this function?

Reply:
Done.